### PR TITLE
DBZ-3134 Adding example for UI

### PR DESCRIPTION
**Pull request by mdrillin**
Adds example for the debezium UI.  @indraraj - I opened a different PR which incorporates comments indicated on the other PR, moves to ui-demo folder, adds screen shots etc.

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Nice one, thanks! Leaving it to @indraraj to take another look and merge.

File: ui-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+You can examine change events which occur due to database changes
+
+```shell
+# Open in a new terminal
+# Viewing the change events in the kafka container
+docker exec -it kafka bash
+
+./bin/kafka-console-consumer.sh --bootstrap-server $BOOTSTRAP_SERVERS --topic [TOPIC_NAME] --from-beginning
+
+```
```
Comment (indraraj) [inline ui-demo/README.md:97]: I think we should also add the cleanup( `docker-compose down`) section at the end.

**Comment by indraraj** [comment]
Thanks, @mdrillin  for taking it forward. Looks good, you can merge it after adding the cleanup section.

**Review by indraraj — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Excellent! Could one of you also look into writing a quick blog post
about the UI? Wouldn't have to be long, explaining the general usage,
some screenshots, etc. @rk3rn3r and @mdrillin, perhaps you can take
the lead on this one? I'd expect it to be similar to what's planned
for docs, but the post could contextualize a bit more and is generally
better in terms of sharing it on social media. Thanks!

File: ui-demo/README.md (hunk 1 above)
Comment (mdrillin) [reply ui-demo/README.md:97]: Yeah, good idea.  Added final section

**Comment by mdrillin** [comment]
Added the cleanup section - merging.

**Pull request merged**